### PR TITLE
install tools for integrations tests in docker image

### PR DESCRIPTION
**Pull request by Lagovas**
* updated ci-py-go docker image that installs dependencies for integration tests
* increase sleep time for censor's tests due to failures in EE CI runs (maybe private not paid repos get less resources in comparison with open-source repos...)

p.s. updated image will be uploaded to docker hub soon

## Checklist

- [x] Change is covered by automated tests
- [x] The [coding guidelines] are followed
- [x] Public API has proper documentation in the [Acra documentation] site or has PR on [documentation repository] 
  with new changes
- [x] CHANGELOG.md is updated (in case of notable or breaking changes)
- [x] CHANGELOG_DEV.md is updated
- [x] Benchmark results are attached (if applicable)
- [x] [Example projects and code samples] are up-to-date (in case of API changes) 

[coding guidelines]: https://golang.org/doc/effective_go
[Example projects and code samples]: https://github.com/cossacklabs/acra-engineering-demo
[Acra documentation]: https://docs.cossacklabs.com/
[documentation repository]: https://github.com/cossacklabs/product-docs

### CODE REVIEW

File: acra-censor/common/logging_logic_test.go [hunk 1]
```diff
@@ -16,7 +16,7 @@ const testSerializationTime = 100 * time.Millisecond
 
 // sleepTime provide extra time to serialize in background goroutine before check
 // default value that will be changed for tests in init function
-const sleepTime = testSerializationTime * 2
+const sleepTime = testSerializationTime * 3
```
Comment (vixentael) [inline acra-censor/common/logging_logic_test.go:19]: 👀

**Review by vixentael — COMMENTED** [review]
👀

File: acra-censor/common/logging_logic_test.go (hunk 1 above)
Comment (Lagovas) [reply acra-censor/common/logging_logic_test.go:19]: actually, it's a weird situation. it works fine on our acra-ce repo (here) and often fail on acra-ee that runs the same tests ) so, I guess it depends on resource shaping for private repositories when you use free credits. but it's just my opinion...

Comment (vixentael) [reply acra-censor/common/logging_logic_test.go:19]: yeah, might be

**Review by ZhmakaAS — APPROVED** [review]

**Review by iamnotacake — APPROVED** [review]

**Pull request merged**